meta-linaro-mbl/meta-optee/recipes-security/optee/optee-os_%.bbappend

```diff
@@ -8,12 +8,11 @@ DEPENDS_append_raspberrypi3-mbl = " arm-aarch64-toolchain-native "
 LICENSE = "BSD-2-Clause"
 
 # MBL_OPTEE_VERSION should be updated to match version pointed to by SRCREV
-MBL_OPTEE_VERSION = "3.4.0"
+MBL_OPTEE_VERSION = "3.6.0"
```

Comment:
Since we are using the upstream repo I think we can drop this variable.
@simonqhughes, what do you think?